lib/Resource/DirectoryUser.php

```diff
@@ -14,9 +14,25 @@ class DirectoryUser extends BaseWorkOSResource
         "rawAttributes",
         "customAttributes",
         "firstName",
+        "email",
+        /**
+         * @deprecated Will be removed in a future major version.
+         * Enable the `emails` custom attribute in dashboard and pull from customAttributes instead.
+         * See https://workos.com/docs/directory-sync/attributes/custom-attributes/auto-mapped-attributes for details.
+         */
```

Comment:
It seems for PHP, you prefix the comment with `[Deprecated]` instead of `@deprecated`

Reply:
Oops, will update, thanks!